homeassistant/components/starlink/binary_sensor.py

```diff
@@ -58,6 +58,12 @@ def is_on(self) -> bool | None:
 
 
 BINARY_SENSORS = [
+    StarlinkBinarySensorEntityDescription(
+        key="update",
+        name="Update available",
+        device_class=BinarySensorDeviceClass.UPDATE,
```

Comment:
We prefer update entities over a binary sensor with this device class.

https://developers.home-assistant.io/docs/core/entity/update

It requires knowing versions installed and available though. Do we know those here?

Reply:
We only know the current version, which is why the binary sensor is back 😛

Reply:
Ok.